scripts/create_release_pr.sh

```diff
@@ -2,16 +2,22 @@
 set -euo pipefail
 
 # Check if there's an existing release PR
-existing_pr=$(gh pr list --state open --search "(Release OR release) in:title" --json url,title --jq '.[0]')
+existing_pr_url=$(gh pr list --state open --base production --head master --json url --jq '.[0].url')
 
-if [ -n "$existing_pr" ]; then
-    echo "Found existing release PR: $(echo "$existing_pr" | jq -r '.url')"
+if [ -n "$existing_pr_url" ]; then
+    echo "Found existing release PR: $existing_pr_url"
     exit 0
 fi
 
 # Create PR title
 today=$(date +%Y-%m-%d)
-release_count=$(gh pr list --state all --search "Release $today in:title" | grep "Release $today" |wc -l)
+# We have to count the number of releases using grep because the GitHub doesn't
+# support searching for exact matches in titles. As a result, the search may return PRs
+# with titles that contain similar dates.
+release_count=$(
+  gh pr list --state merged --base production --head master --search "$today in:title" --json title --jq '.[].title' |
+  grep -c "$today"
+)
```

Comment:
```suggestion
release_count=$(
  gh pr list --state merged --base production --head master --search "$today in:title" --json title --jq '.[].title' |
  grep -c "Release $today" || echo "0"
)
```
 If no PRs match the date search, grep -c "$today" will exit with status 1 (because of set -e), causing the script to fail.

This is the main issue I resolved in my PR. The script is still failing (on this branch).

Let's also keep the word boundary "Release ...". This is more robust.

Reply:
> If no PRs match the date search, grep -c "$today" will exit with status 1 (because of set -e), causing the script to fail.

My test probably went off, will fix.

> Let's also keep the word boundary "Release ...". This is more robust.

Technically, any PR from master based on production (and arguably any PR based on production) is a release PR. So, I think it's actually better to remove "Release." It simplifies the script and makes it less dependent on the title.